docs/concepts/storage/persistent-volumes.md

```diff
@@ -79,8 +79,8 @@ When a user is done with their volume, they can delete the PVC objects from the
 The Retain reclaim policy allows for manual reclamation of the resource. When the `PersistentVolumeClaim` is deleted, the `PersistentVolume` still exists and the volume is considered "released". But it is not yet available for another claim because the previous claimant's data remains on the volume. An administrator can manually reclaim the volume with the following steps.
 
 1. Delete the `PersistentVolume`. The associated storage asset in external infrastructure (such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume) still exists after the PV is deleted.
-1. Manually clean up the data on the associated storage asset accordingly.
-1. Manually delete the associated storage asset, or if you want to reuse the same storage asset, create a new `PersistentVolume` with the storage asset definition.
+2. Manually clean up the data on the associated storage asset accordingly.
```

Comment:
the number doesn't matter actually, markdown will figure out the correct sequence.
in fact, always using '1' would help alleviate the workload on managing the sequence manually.

Reply:
@tengqm Okay, I revert the number changes.